src/Uno.UI/UI/Xaml/Controls/Border/Border.iOSmacOS.cs

```diff
@@ -56,7 +57,13 @@ private void UpdateBorderLayer(_Image backgroundImage = null)
 			{
 				if (backgroundImage == null)
 				{
-					(Background as ImageBrush)?.ImageSource?.TryOpenSync(out backgroundImage);
+					ImageData backgroundImageData = default;
+					(Background as ImageBrush)?.ImageSource?.TryOpenSync(out backgroundImageData);
```

Comment:
Method name `TryOpenSync` indicates a failure possibility, in this case `backgroundImageData` might be null. Then we dereference `backgroundImageData` below. Are we missing a test case?

Otherwise consider something like:

```csharp
if ((Background as ImageBrush)?.ImageSource?.TryOpenSync(out backgroundImageData) == true)
{
    // use backgroundImageData here
}
```

Reply:
`backgroundImageData` will never be null (it is a `struct` and the variable is not declared as nullable), but I have adjusted the `if` statement to be more readable 👍